app/controllers/prison_staff_application_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class PrisonStaffApplicationController < PrisonsApplicationController
```

Comment:
If I understand the intention of this class correctly, I wonder if a name like `PomApplicationController` would be more descriptive?

From what I can tell: this is an 'application controller' for actions related to the currently signed in POM.

The name "prison staff" is a bit misleading because SPOs/HOMDs are also members of prison staff.

Reply:
Ah – ignore me. Looks like the "prison staff" name comes from the routing structure `/prisons/:prison_id/staff/:staff_id`, so I guess it makes sense within that context.

Still – it seems like we use "POM" and "staff" interchangeably, which is a little confusing. But not something to be resolved in this PR.